post_installation_configuration/node-tasks.adoc

```diff
@@ -19,6 +19,25 @@ include::modules/rhel-adding-node.adoc[leveloffset=+2]
 include::modules/rhel-ansible-parameters.adoc[leveloffset=+2]
 include::modules/rhel-removing-rhcos.adoc[leveloffset=+2]
 
+[id="post-install-config-adding-fcos-compute"]
+== Adding {op-system-first} compute machines to an {product-title} cluster
+
+You can add more compute machines to your {product-title} cluster on bare metal.
+
+Before you add more compute machines to a cluster that you installed on bare metal infrastructure, you must create {op-system} machines for it to use. Follow either of the steps to use an ISO image or network PXE booting to create the machines.
```

Comment:
I know this wording was already there (from another module), but the wording "Follow either of the steps" sounds weird to me. I'd suggest something like "You can either use an ISO image or network PXE booting to create the machines." or something similar.